lib/PuppeteerSharp.Tests/BrowserContextTests/DefaultBrowserContextTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace PuppeteerSharp.Tests.BrowserContextTests
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class DefaultBrowserContextTests : PuppeteerPageBaseTest
+    {
+        public DefaultBrowserContextTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task PageCookiesShouldWork()
```

Comment:
What are you exactly testing here? Seems pretty similar to `PageSetCookieShouldWork` except you are setting the cookie with a piece of JS, right? What's the difference exactly?
Not saying it's wrong, maybe the name can be adjusted?